charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -576,9 +576,9 @@ sscs-evidence-share:
 # click create
 # azure steps should be repeated every time PR is undeployed from preview
 sscs-tya-notif:
-  enabled: false
+  enabled: true
   java:
-    image: hmctspublic.azurecr.io/sscs/tya-notif:latest
+    image: hmctspublic.azurecr.io/sscs/tya-notif:pr-2417
```

Comment:
For future, worth not committing any changes to preview values unless it's to keep the defaults updated / correct. Should leave as latest for images, etc.

Reply:
removed this